README.rst

```diff
@@ -0,0 +1,203 @@
+Required packages
+-----------------
+
+To run the PyPI software, you need Python 2.7+ and PostgreSQL
+
+
+Quick development setup
+-----------------------
+
+Make sure you read http://wiki.python.org/moin/CheeseShopDev#DevelopmentEnvironmentHints
```

Comment:
Can you remove/update this line it links to a page that states that it is immediately out of date and that people should come back here to learn what to do. Infinite loops are sad.